Neos.Flow/Classes/ObjectManagement/DependencyInjection/ProxyClassBuilder.php

```diff
@@ -442,13 +442,6 @@ protected function buildPropertyInjectionCodeByConfiguration(Configuration $obje
     public function buildPropertyInjectionCodeByString(Configuration $objectConfiguration, ConfigurationProperty $propertyConfiguration, $propertyName, $propertyObjectName)
     {
         $className = $objectConfiguration->getClassName();
-
-        if (strpos($propertyObjectName, '.') !== false) {
-            $settingPath = explode('.', $propertyObjectName);
-            $settings = Arrays::getValueByPath($this->configurationManager->getConfiguration(ConfigurationManager::CONFIGURATION_TYPE_SETTINGS), array_shift($settingPath));
-            $propertyObjectName = Arrays::getValueByPath($settings, $settingPath);
-        }
```

Comment:
This is ancient (~7 year old) magic that turned an inject class name into a setting if it contained a "."..
With the introduction of the `InjectConfiguration` annotation this should have been removed, but it was probably forgotton